zlibWrapper/gzwrite.c

```diff
@@ -64,6 +64,8 @@ local int gz_init(gz_statep state) {
         strm->next_out = state.state->out;
         state.state->x.next = strm->next_out;
     }
+
+    free(state.state);
```

Comment:
That one is weird.
I'm not even sure what's going on in this code.

One of the first actions in this function is : `state.state->in = (unsigned char*)malloc(state.state->want << 1);`,
which presumes that `state.state` is already allocated (should probably be asserted).
which means that, something else has allocated `state.state`, and therefore something else is in charge of freeing it.
I don't see how it could be good to `free` it here.

Reply:
Apologies, You're right. That was an oversight from me.
I'll fix this tomorrow morning... Again, sorry for the inconvenience